src/postgres-util/src/lib.rs

```diff
@@ -284,10 +287,15 @@ impl Config {
                 Ok(client)
             }
             SshTunnelConfig::Tunnel { .. } => {
+                let (session, local_port) = self.establish_openssh_connection().await?;
                 let tls = MakeTlsConnect::<TokioTcpStream>::make_tls_connect(&mut tls, &self.host)?;
-                let channel = self.establish_ssh_connection().await?;
-                let (client, connection) = self.postgres_config.connect_raw(channel, tls).await?;
-                task::spawn(|| connection_task_name, connection);
+                let tcp_stream = TokioTcpStream::connect(("localhost", local_port)).await?;
+                let (client, connection) =
+                    self.postgres_config.connect_raw(tcp_stream, tls).await?;
+                task::spawn(|| connection_task_name, async {
```

Comment:
we don't need to use `connect_raw` anymore right? we can just use `connect` with the right port and host?

Reply:
Ah, this was mostly because I'd have to recreate a `postgres_config` from scratch because I can't just patch the hostname there to point to `localhost`. Lemme think a bit more about it to see if I can find a clean way to do this.

Reply:
https://docs.rs/tokio-postgres/latest/tokio_postgres/config/struct.Config.html#method.host should work right?

Reply:
oh wait that pushes to a list, thats annoying

Reply:
Yeah, it still tries to the original address, and only after it fails for real (dozens of seconds later) it goes to the next one.

Reply:
ugh, yeah then just a `TODO` here is fine (cause we can improve this in our fork at some point)